sdk/data/azcosmos/emulator_cosmos_global_endpoint_manager_test.go

```diff
@@ -30,6 +30,7 @@ func TestGlobalEndpointManagerEmulator(t *testing.T) {
 		ReadRegions:                  []accountRegion{emulatorRegion},
 		WriteRegions:                 []accountRegion{emulatorRegion},
 		EnableMultipleWriteLocations: false,
+		AccountConsistency:           userConsistencyPolicy{"Strong"},
```

Comment:
nit: Are there any tests where we are checking emitted logs that need to be updated?

Reply:
nope - just the one test I had to update on verifying account settings for the emulator